tests/customqf.vader

```diff
@@ -0,0 +1,92 @@
+Include: include/setup.vader
+
+Execute (Displays custom formatted quickfix buffer):
+  call neomake#quickfix#enable()
+  call setloclist(0, [{
+      \   'lnum': 1,
+      \   'bufnr': 1,
+      \   'text': 'Line 1 nmcfg:{"name": "Alpha", "short": "Alph"}',
+      \ },
+      \ {
+      \   'lnum': 2,
+      \   'col': 5,
+      \   'bufnr': 1,
+      \   'text': 'Line 2',
+      \ }])
+
+  lopen
+
+  AssertEqual getline(1, '$'),
+    \ ['Alph 1:- Line 1 ', 'Alph 2:5 Line 2']
+
+  normal! 0
+  " :normal! ignores autocmd's here?
+  doautocmd CursorMoved
```

Comment:
Not supported by Vader apparently: https://github.com/junegunn/vader.vim#some-events-may-not-be-triggered - doing it manually is the way to go.